docs/current-1.1/contributing.md

```diff
@@ -0,0 +1,60 @@
+# Using the current-1.1 folder and sub-folders
+
+This folder is the top-level node that matches the [docs](../index.md)
```

Comment:
This now links to `docs/index.md`, which is still a broken link. Should it link to `welcome.md`?